CHANGELOG.md

```diff
@@ -1,27 +1,222 @@
+    - Whether to enable user behavior analytics.
+    - Desktop version now supports memory size configuration through JVM parameters.
+- Added database availability indicators; the database list under projects will now show unavailable statuses and reasons.
+- Improved the initiation interaction for tickets:
+    - Support for initiating various tickets directly from the database in the resource tree.
+    - Commonly used tickets (mock data, database changes, data archiving, data cleaning, database permission application) support reinitiation with editable task parameters.
+- Enhanced database change processes to detect index change statements and automatically adjust the timeout setting (120h) to prevent index change statement execution failure due to timeout.
+- Desktop version personal settings now support custom JVM configuration with memory usage control to within 1 GB.
+- Desktop version supports exporting data files larger than 4 GB.
+- Optimized the en-US language wording of the product.
+
+### Bug fixes
+
+Connection Session
+
+- Sessions failing to remove after expiration, causing potential leaks [#2125](https://github.com/oceanbase/odc/pull/2125)
+- References to expired connection sessions failed to be promptly cleared, leading to resource leaks, which may cause increased memory consumption [#1914](https://github.com/oceanbase/odc/pull/1914)
+- Under high-frequency usage scenarios, executing SQL or viewing table data may encounter issues where the interface becomes unresponsive [#1797](https://github.com/oceanbase/odc/pull/1797)
+- Intermittent 404 errors when attempting to open SQL console [#1809](https://github.com/oceanbase/odc/pull/1809)
+
+SQL Execution
+
+- Commit/rollback button state not in sync with actual transaction status in OceanBase 4.2 [#2097](https://github.com/oceanbase/odc/pull/2097)
+- SQL statements with single-line comments unable to execute [#2085](https://github.com/oceanbase/odc/pull/2085)
+- Incorrect offsets for last SQL command without a trailing delimiter [#1970](https://github.com/oceanbase/odc/pull/1970)
+- Incompatibility of anonymous block execution module with Oracle 11g [#1759](https://github.com/oceanbase/odc/pull/1759)
+- DBMS output failed to output spaces [#1051](https://github.com/oceanbase/odc/issues/1970)
+- `#` and `$` character disappeared after SQL window formatted the code [#1490](https://github.com/oceanbase/odc/issues/1490)
+- Auto-complete in SQL window not available for MySQL datasource [#1718](https://github.com/oceanbase/odc/issues/1718)
+
+Result-set
+
+- Degrading performance when modifying multiple rows concurrently in result-set [#2007](https://github.com/oceanbase/odc/pull/2007)
+- In OceanBase MySQL mode, the datetime data type loses precision when displayed [#1996](https://github.com/oceanbase/odc/pull/1996)
+- Switching between viewing BLOB field text and hexadecimal images may cause the UI to freeze [#300](https://github.com/oceanbase/odc/issues/300)
```

Comment:
- Modifying multiple rows of data simultaneously in the result set takes a long time.
- Precision loss occurs when displaying DATETIME data type in OceanBase MySQL mode.
- Switching back and forth between viewing BLOB field text and hexadecimal images may cause interface freezing.